resources/netlog_defs.ts

```diff
@@ -342,6 +344,9 @@ const latestLogDefinitions = {
       z: 11,
       heading: 12,
     },
+    possibleRsvFields: {
+      ability: 5,
```

Comment:
Is the name here ever used? I think this should probably be either `'ability'` or `'5'` and not redundantly both?

Reply:
> Is the name here ever used? I think this should probably be either `'ability'` or `'5'` and not redundantly both?

Nah, it's not used.  I did that to maintain the same structure as `fields`, and so it didn't look like a magic-number situation (also easier to visually see what fields we'll be decoding without having to look at `fields` -- although I suppose a comment would also accomplish the same thing.